web/themes/custom/digital_gov/digital_gov.theme

```diff
@@ -386,3 +387,40 @@ function digital_gov_theme_suggestions_container_alter(array &$suggestions, arra
     $suggestions[] = 'container__no_wrapper';
   }
 }
+
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function digital_gov_preprocess_node__landing_page__job_board(&$variables, $hook) {
+  $filtered_array = [];
+  $entity_type_manager = \Drupal::entityTypeManager();
+  $config_pages = $entity_type_manager->getStorage('config_pages')->load('job_board');
+  if (isset($config_pages->get('field_job_board_yml_contents')->getValue()[0]['value'])) {
+    $yml_string = $field_description = $config_pages->get('field_job_board_yml_contents')->getValue()[0]['value'];
```

Comment:
$field_description is not used, I think it can be removed.
FYI:
`isset($config_pages->get('field_job_board_yml_contents')->getValue()[0]['value'])` will work, but 
`$config_pages->get('field_job_board_yml_contents')->isEmpty()` will work in the case of ALL field types.
Also, `$config_pages->get('field_job_board_yml_contents')->getString()` is a really handy shortcut if the cardinality of the field is always 1.